### ISSUE TITLE
Option to wrap $var names in curly brackets for readability

### ISSUE THREAD

**Issue by Aquaricat** [UnicornHeartClub/Issues#63]
In #60 there's an example for the Toll the Bell spell that uses some theoretical eventual capability.
```
$dice = #cantrip
$die = #toll_hp_check
!r $diced$die
```

This is actually really hard to read.  It'd be better if I could wrap the `${var}` name somehow to make it a bit easier.
```
${dice}d${die}
```

**Comment by Pjb518** [comment]
I think that is a fantastic addition. I'm not actually certain the original version would even roll right now, because it would require a space.

**Comment by Aquaricat** [comment]
It could theoretically be done with some really hideous regex, but... yeah, good chance it wouldn't.

**Comment by Pjb518** [comment]
Having just tested it, I can tell you that is produces a consistent output of zero, and attempts to click the value to see the roll process causes the app to crash.

**Comment by Aquaricat** [comment]
I'm just trying to think how you'd even catch it with Regex.  Search for either a number, or a $ proceeded by a d, then look back at everything before the d.  Like I said, hideous!  ^^

**Comment by Pjb518** [comment]
Fixed in 0.4.3, and this does allow rolls such as the one below to function correctly.

```bash
$var = 5
$var2 = 6
!r ${var}d${var2}
```
